packages/video_player/video_player_android/android/src/test/java/io/flutter/plugins/videoplayer/ExoPlayerEventListenerTest.java

```diff
@@ -56,7 +59,25 @@ public void onPlaybackStateChangedReadySendInitialized() {
   }
 
   @Test
-  public void onPlaybackStateChangedReadyInPortraitMode90DegreesSwapWidthAndHeight() {
+  @Config(sdk = 29)
+  public void
+      onPlaybackStateChangedReadySendInitializedWithRotationCorrectionAndWidthAndHeightSwap_aboveAndroid29() {
+    VideoSize size = new VideoSize(800, 400, 0, 0);
+    int rotationCorrection = 90;
+    Format videoFormat = new Format.Builder().setRotationDegrees(rotationCorrection).build();
+
+    when(mockExoPlayer.getVideoSize()).thenReturn(size);
+    when(mockExoPlayer.getDuration()).thenReturn(10L);
+    when(mockExoPlayer.getVideoFormat()).thenReturn(videoFormat);
+
+    eventListener.onPlaybackStateChanged(Player.STATE_READY);
+    verify(mockCallbacks).onInitialized(400, 800, 10L, rotationCorrection);
+  }
+
+  @Test
+  @Config(sdk = 21)
```

Comment:
consider setting min/max sdk here and in other places where you use only sdk. If a wide range is not appropriate consider setting multiple sdk values to at least expand coverage. 

~Finally if this is not specific to api 21 but to the minimum flutter supported version then consider using  [FLUTTER_MIN](https://github.com/flutter/engine/pull/55002/files#diff-c4aa0dd8e137bba5e617ff29c44d35452536c5b83beb972936e153b2a91e7e29)~
Nvm that was in an engine pr.